x/mint/spec/02_state.md

```diff
@@ -3,7 +3,6 @@ order: 2
 -->
 
 # State
-
 ## LastBlockTime
 
 LastBlockTime defines block time of the last block, It is used to calculate inflation.
```

Comment:
It seems "It" --> "it" is needed for making it with small letter.

Reply:
> It seems "It" --> "it" is needed for making it with small letter.
You mean you lower case for `It`? I will make that change. Thanks for the catch.